account_netting/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_account_move_make_netting,Full access on account.move.make.netting to accountant grp,model_account_move_make_netting,account.group_account_user,1,1,1,1
+access_account_move_make_netting_manager,Full access on account.move.make.netting to accountant manager grp,model_account_move_make_netting,account.group_account_manager,1,1,1,1
```

Comment:
Why do you need this one? The previous line applies also to this group.

Reply:
Doesn't work if you install only this module.

Reply:
AFAIK, `group_account_manager` inherits from `group_account_user`, isn't it?

Reply:
I think that come from `account_usability` module unless I'm mistaken

Reply:
@pedrobaeza What do I do then? Would you prefer that I remove this line of code?

Reply:
Hi @pedrobaeza

Please give us feedback here.

We require this module in v17.0

Reply:
OK, the existing permission was "Technical / Show Full Accounting Features", and this one is adding "Invoicing / Billing Administrator" to do the netting. It's a change of scope, but I think is correct.